package.json

```diff
@@ -9,14 +9,19 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
+    "@types/jest": "^26.0.14",
+    "@types/node": "^14.11.5",
+    "@types/react": "^16.9.51",
+    "@types/react-dom": "^16.9.8",
     "chroma-js": "^2.1.0",
     "immutable": "^4.0.0-rc.12",
     "node-sass": "^4.14.1",
     "normalize.css": "^8.0.1",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
     "react-scripts": "3.4.3",
-    "styled-components": "^5.2.0"
+    "styled-components": "^5.2.0",
+    "typescript": "^3.9.7"
```

Comment:
I'm pulling in Typescript 3.9 instead of 4.0 since it sounds like there's still some compatibility issues between CRA and Typescript 4.0.